about.css

```diff
@@ -0,0 +1,435 @@
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 1rem;
+  background-color: #ec4327;
+  background-blend-mode: multiply;
+}
+
+.year {
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+  font-weight: 700;
+  font-size: 1.5rem;
+  line-height: 1rem;
+}
+
+p.about-summit {
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.75rem;
+}
+
+.partner {
+  width: 100%;
+  padding: 2.2rem 1.5rem;
+  background: #414246;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.partner h2 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  color: #fff;
+  padding: 0.5rem;
+  text-align: center;
+}
+
+.partner-logos {
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  flex-wrap: wrap;
+}
+
+footer {
+  width: 80%;
+  margin: 0 auto;
+  padding: 4rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 10%;
+  background-color: #fff;
+}
+
+footer i {
+  color: #3e3e3e;
+  font-size: 5rem;
+}
+
+@media only screen and (min-width: 768px) {
+  header {
+    position: relative;
+  }
+
+  nav .top {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: flex-end;
+    margin: 0 auto;
+    background: #3e3e3e;
+    padding: 0.2rem 1rem;
+    gap: 1rem;
+  }
+
+  nav .top a {
+    font-family: 'Lato', sans-serif;
+    color: #fff;
+  }
+
+  nav .bottom {
+    width: 90%;
+    display: flex;
+    padding: 0.5rem;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+  }
+
+  nav .bottom .nav-menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .nav-menu li:nth-child(6) {
+    padding: 0.5rem;
+    border: 5px solid #ec4327;
+  }
+
+  .nav-menu li:nth-child(6) a {
+    padding: 0.5rem;
+    color: #ec4327;
+  }
+
+  .nav-link {
+    font-family: 'Lato', sans-serif;
+    color: #868686;
+  }
+
+  .hello {
+    display: none;
+  }
+
+  .hero-about {
+    background-image: url('./images/hero_img.png');
+  }
+
+  .about-hero-wrapper h1 {
+    font-size: 2.5rem;
+    font-weight: bold;
+    line-height: 3rem;
+  }
+
+  .about-description {
+    width: 80%;
+    margin: 0 auto;
+    font-size: 0.9rem;
+    line-height: 1.5rem;
+    padding: 1.5rem 1rem;
+    text-align: center;
+    margin-bottom: 2rem;
+  }
+
+  .partner {
+    display: none;
+  }
+
+  footer {
+    background: #2e2e2e;
+    width: 100%;
+  }
+
+  footer p {
+    color: #fff;
+  }
+
+  footer i {
+    color: #fff;
+    font-size: 7rem;
+  }
+
+  .logo-sub {
+    padding: 0;
+  }
+
+  .d-line {
+    width: 100%;
+    margin-bottom: 1rem;
+  }
+
+  .testimonial {
+    width: 60%;
+    aspect-ratio: 2/1;
+  }
+
+  .all-testimonials {
+    width: 80%;
+    margin: 0 auto;
+    flex-direction: row;
+  }
```

Comment:
- [ ] You have done a good job by reducing the width of these cards 👍👍 However, kindly reduce the width a bit more to match the design guidelines as suggested [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180711126)
- [OPTIONAL] What do you think of adding some space/padding/margin to the top and bottom of these cards as mentioned in the following images? 💭 I think it will enhance your design ✨

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235278059-b5cbeb63-5a25-434a-9896-f2cc09b3a660.png) | ![image](https://user-images.githubusercontent.com/106119105/235278033-38dc6975-be1f-41b4-9e5e-cb00249dba1f.png)